crates/bevy_input/src/keyboard.rs

```diff
@@ -318,16 +318,19 @@ pub enum KeyCode {
     /// The `Kanji` key.
     Kanji,
 
-    /// The `LAlt` / `Left Alt` key. Maps to `Left Option` on Mac.
-    LAlt,
-    /// The `LBracket` / `Left Bracket` key.
-    LBracket,
-    /// The `LControl` / `Left Control` key.
-    LControl,
-    /// The `LShift` / `Left Shift` key.
-    LShift,
-    /// The `LWin` / `Left Windows` key. Maps to `Left Command` on Mac.
-    LWin,
+    /// The `Left Alt` key. Maps to `Left Option` on Mac.
+    AltLeft,
```

Comment:
Very minor thing: The doc string is now the reverse of the actual name.
Not sure if this matters though.

Reply:
Yeah I'm not a fan of winit's naming here but it makes sense to be consistent with them

Reply:
Yeah I think this is fine.